zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/ir.kt

```diff
@@ -404,3 +404,15 @@ fun getOrCreateCompanion(
   enclosing.declarations.add(companionClass)
   return companionClass
 }
+
+fun FqName.classId(name: String): ClassId {
+  return ClassId(this, Name.identifier(name))
+}
+
+fun FqName.callableId(name: String): CallableId {
```

Comment:
I think this would only work when the` FqName` is a package, and would not work for classes. Would it be worth defining a value class `FqPackageName` and then defining this function on that class🤔.

Reply:
I think it makes sense. Updated :+1: